File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height >= 9);

    for (int i = 0; i < height; i++)
    {
        //Print spaces
        for (int j = height - i - 1; j > 0; j--)
        {
            printf(" ");
        }
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        printf("  ");
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: sentimental-mario-more/mario.py

from cs50 import get_int


def print_spaces(num):
    for i in range(num):
        print(" ", end='')


def print_hashtag(num):
    for i in range(num):
        print("#", end='')


n = 0
while n < 1 or n > 8:
    n = get_int("Height: ")

for i in range(n):
    print_spaces(n - i - 1)
    print_hashtag(i + 1)
    print("  ", end='')
    print_hashtag(i + 1)
    print("")

